manifests/cf-manifest/spec/secret_generation_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'tempfile'
+
+RSpec.describe "secret generation" do
+  describe "generate-cf-secrets" do
+    let(:script) {
+      File.expand_path("../../scripts/generate-cf-secrets.sh", __FILE__)
+    }
+
+    specify "it should produce lint-free YAML" do
+      pending "No mkpasswd(1) on Mac OS X" if Gem::Platform.local.os == 'darwin'
```

Comment:
`skip` would be better than `pending` here. `pending` causes rspec to still run the test, and actually flag it as a failure if it passes (ie, by using `pending` you're effectively adding an assertion that the test fails). `skip` on the other hand, just skips this test. See http://www.relishapp.com/rspec/rspec-core/v/3-4/docs/pending-and-skipped-examples for more details.